tools/rtpsrelay/control/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.23...4.0)
+project(opendds_rtps_relay_control CXX)
+
+set(dep_libs
+  OpenDDS::Rtps_Udp
+  OpenDDS::Security
+  OpenDDS::RtpsRelayLib
+)
+find_package(OpenDDS REQUIRED ${dep_libs} built_in_topics safety_profile=FALSE)
+include(opendds_build_helpers)
+
+add_executable(RtpsRelayControl
+  RtpsRelayControl.cpp
+)
+_opendds_executable(RtpsRelayControl)
```

Comment:
Since it's installed (I'm assuming that's what's intended), can you added it here with the other installed executables?

https://github.com/OpenDDS/OpenDDS/blob/6ff532437e50d7d097a11a502a959d844b21ff48/cmake/opendds_group.cmake#L66-L68

I'm not sure who'd want to do this, but it allows users to require it, for example: `find_package(OpenDDS REQUIRED OpenDDS::RtpsRelayControl)` and install the executable explicitly.